src/main/java/feedmysheep/feedmysheepapi/domain/cell/app/repository/CellGatheringRepository.java

```diff
@@ -18,4 +22,16 @@ public interface CellGatheringRepository extends JpaRepository<CellGatheringEnti
   @Query("SELECT cg FROM CellGatheringEntity cg WHERE cg.cellGatheringId = :cellGatheringId and cg.isValid = true")
   CellGatheringEntity getCellGatheringByCellGatheringId(
       @Param("cellGatheringId") Long cellGatheringId);
+
+  // TODO 테스트코드 작성
+  @Transactional
+  @Modifying
+  @Query("UPDATE CellGatheringEntity cg SET cg.gatheringTitle = :#{#updateDto.gatheringTitle}, cg.gatheringDate = :#{#updateDto.gatheringDate}, cg.startedAt = :#{#updateDto.startedAt}, cg.endedAt = :#{#updateDto.endedAt}, cg.gatheringPlace = :#{#updateDto.gatheringPlace}, cg.gatheringPhotoUrl = :#{#updateDto.gatheringPhotoUrl}, cg.description = :#{#updateDto.description}, cg.leaderComment = :#{#updateDto.leaderComment}, cg.pastorComment = :#{#updateDto.pastorComment} WHERE cg.cellGatheringId = :cellGatheringId and cg.isValid = true")
+  void updateCellGatheringId(
+      @Param("updateDto") CellReqDto.updateCellGathering updateDto,@Param("cellGatheringId") Long cellGatheringId);
+
+  // TODO 테스트코드 작성
+  @Query("SELECT cg FROM CellGatheringEntity cg WHERE cg.cellGatheringId = :cellGatheringId and cg.isValid = true")
+  Optional<CellGatheringEntity> getOptionalCellGatheringByCellGatheringId(
+      @Param("cellGatheringId") Long cellGatheringId);
```

Comment:
```suggestion
  @Query("SELECT cg FROM CellGatheringEntity cg WHERE cg.cellGatheringId = :cellGatheringId and cg.isValid = true")
  Optional<CellGatheringEntity> getCellGatheringByCellGatheringId(
      @Param("cellGatheringId") Long cellGatheringId);
```
메서드 명에 Optional은 빼주세요.